packages/commonwealth/client/scripts/views/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,176 @@
+import 'components/Breadcrumbs/Breadcrumbs.scss';
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { CWText } from '../component_kit/cw_text';
+import { CWTooltip } from '../component_kit/cw_popover/cw_tooltip';
+import app from 'state';
+
+export const Breadcrumbs = () => {
+  const location = useLocation();
+
+  const user = app.user.addresses[0];
+  const profileId = user?.profileId || user?.profile.id;
+
+  let standalone = false;
+
+  /**
+   * An array of paths that are considered standalone pages.
+   */
+
+  const standalonePaths: Array<string> = [
+    'explore',
+    'dashboard',
+    'notifications',
+    'notification-settings',
+    'createCommunity',
+  ];
+
+  /**
+   * Checks if the current page is a standalone page or if it contains the profileId.
+   *
+   * @param {string} pathname - The pathname of the current location.
+   * @param {string} profileId - The profile ID to check for.
+   * @returns {boolean} True if the page is standalone or contains the profileId, otherwise false.
+   */
+  if (
+    standalonePaths.includes(location.pathname.split('/')[1]) ||
+    location.pathname.includes(String(profileId))
+  ) {
+    standalone = true;
+  }
+  /**
+   * Parses the current location pathname into an array of breadcrumb items.
+   *
+   * @param {Object} location - The location object containing pathname information.
+   * @returns {Array} An array of breadcrumb items, each with a text, link, and isParent property.
+   */
+  const pathnames = location.pathname
+    .split('/')
+    .map((x, index, arr) => {
```

Comment:
dont use `x` or other non meaningfully names for variables

Reply:
This has been finished. (Making this note to keep progress)